### DISCUSSION TITLE
节点状态在线但是用不了

### DISCUSSION THREAD

**Discussion by takumijie** [natfrp/wiki#57] [使用问题] [answered]
我现在使用的是#香港CN2-NF的节点用于网站内穿，但是今天突然连不上了。

[a comment by an automated account is not shown]

**Reply by EvanNotFound** [reply]
什么时候能好呢

**Reply by pilipala233** [reply]
同问。。

**Reply by fakeboboliu** [reply]
> 什么时候能好呢

Well，在有人攻击的时候我们的想法是没有意义的，什么时候服务器能恢复只能取决于攻击者打算什么时候收手。

另外：更换高防服务器虽然是解决方案，但是因为我们主要服务用户是中国境内用户，对境外节点的优先级较低，加上上海疫情拉闸了上新节点的物流进程，我们短时间内没有更换的计划。

Edit：短时间过去了，我们换了个 ip，虽然没有高防但是起码现在能用了（

**Reply by EvanNotFound** [reply]
行，问题不大，我用别的节点
